test-violations.py

```diff
@@ -0,0 +1,62 @@
+from attrs import define
+from attrs import field
+
+
+@attr.s(auto_attribs=True)
+class ExampleClass(object):
+    # ruleid:attr-mutable-initializer
+    empty_dict = {}
+    # ruleid:attr-mutable-initializer
+    empty_list = []
```

Comment:
:exclamation:**Cycode**: SAST violation: 'Unsafe usage of mutable initializer with attr.s decorator'.
Severity: Medium <!-- d7e14281-4a66-4c71-9ca2-e8fff8203654 -->